node/flows.re

```diff
@@ -412,3 +414,51 @@ let request_withdraw_proof = (state, ~hash) =>
   };
 let request_ticket_balance = (state, ~ticket, ~address) =>
   state.Node.protocol.ledger |> Ledger.balance(address, ticket);
+
+let trusted_validators_membership =
+    (
+      ~file,
+      ~persist:
+         (list(Trusted_validators_membership_change.t), ~file: string) =>
+         Lwt.t(unit),
```

Comment:
```suggestion
      ~persist,
```

Reply:
The typechecker has difficult infering types here

Reply:
Oh that's a neat case, possible improvements, you can reduce it to `~persist: (_, ~file: _) => _` to indicate the source of the problem or just revert the call site below from `persist(~file, param)` to `persist(param, ~file)`.

Reply:
Addressed this by changing the order of params in the call site
This was interesting indeed.